src/workerd/api/worker-rpc.c++

```diff
@@ -70,14 +70,15 @@ kj::Maybe<jsg::JsValue> WorkerRpc::getNamed(jsg::Lock& js, kj::StringPtr name) {
   // Named intercept is enabled, this means we won't default to legacy behavior.
   // The return value of the function is a promise that resolves once the remote returns the result
   // of the RPC call.
-  return jsg::JsValue(js.wrapReturningFunction(js.v8Context(), [this, methodName=kj::str(name)]
-      (jsg::Lock& js, const v8::FunctionCallbackInfo<v8::Value>& args) {
+  return jsg::JsValue(js.wrapPromiseReturningFunction(js.v8Context(),
```

Comment:
This diff isn't actually needed to fix the bug, right?

Reply:
No but it's a good cleanup and helped isolate the issue a bit more by removing one of the factors. I prefer to keep this in.